include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -422,19 +422,19 @@ typedef enum OrtMemType {
   OrtMemTypeDefault = 0,                // the default allocator for execution provider
 } OrtMemType;
 
-ORT_API_STATUS(OrtCreateAllocatorInfo, _In_ const char* name1, enum OrtAllocatorType type, int id1, enum OrtMemType mem_type1, _Outptr_ OrtMemoryInfo** out);
+ORT_API_STATUS(OrtCreateMemoryInfo, _In_ const char* name1, enum OrtAllocatorType type, int id1, enum OrtMemType mem_type1, _Outptr_ OrtMemoryInfo** out);
 
 /**
- * Convenience function for special case of OrtCreateAllocatorInfo, for the CPU allocator. Uses name = "Cpu" and id = 0.
+ * Convenience function for special case of OrtCreateMemoryInfo, for the CPU allocator. Uses name = "Cpu" and id = 0.
  */
-ORT_API_STATUS(OrtCreateCpuAllocatorInfo, enum OrtAllocatorType type, enum OrtMemType mem_type1, _Outptr_ OrtMemoryInfo** out)
+ORT_API_STATUS(OrtCreateCpuMemoryInfo, enum OrtAllocatorType type, enum OrtMemType mem_type1, _Outptr_ OrtMemoryInfo** out)
 ORT_ALL_ARGS_NONNULL;
 
 /**
  * Test if two allocation info are equal
```

Comment:
>allocation info [](start = 15, length = 15)

memory info?

Reply:
didn't follow.

Reply:
the comment is a bit confusing referring to 'allocation info' when all the params are about memory info. was wondering if different wording would help given this is a public API 

---
In reply to: [323076249](https://github.com/microsoft/onnxruntime/pull/1804#discussion_r323076249) [](ancestors = 323076249)